WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m

```diff
@@ -379,6 +379,9 @@ - (WPAnalyticsTrackerMixpanelInstructionsForStat *)instructionsForStat:(WPAnalyt
             [instructions setSuperPropertyAndPeoplePropertyToIncrement:@"number_of_times_editor_published_post"];
             [instructions setCurrentDateForPeopleProperty:@"last_time_published_post"];
             break;
+        case WPAnalyticsStatPushNotificationAlertPressed:
+            instructions = [WPAnalyticsTrackerMixpanelInstructionsForStat mixpanelInstructionsForEventName:@"PushNotification - Alert Tapped"];
```

Comment:
@sendhil sure! Updated [in this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/3430/files)